### DISCUSSION TITLE
Kum saatleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#109] [Zeka Sorusu]
elimizde 7 ve 11 dakikalık iki adet kum saati var. bu iki kum saatini kullanarak dolapta soğumuş yemeği ocakta 1 dakika nasıl ısıtırız?

not: başka saatimiz yok, aklımızdan da saniye saymıyoruz

**Comment by adaakoluk** [comment]
7 dakikalık kum saatini 7 kere çevirip 11 dakikalık kum saatini 1 kere çevirdiğimizde yemeği 1 dakika ısıtmış oluruz.

**Reply by sonsuzus** [reply]
3 sefer çevirsek 21, 11 lik de iki defa çevrilse 22 sanki bir dakikalık farkı daha erken yakalayabiliriz gibi?
